jupyter/Dockerfile

```diff
@@ -1,9 +1,13 @@
 # Image: pubnative/pyspark-ci
 # Cf: Makefile.
 
-FROM jupyter/all-spark-notebook:7d427e7a4dde
+FROM jupyter/all-spark-notebook:703d8b2dcb88
 
-LABEL maintainer Denis <denis@pubnative.net>
+LABEL maintainer=datascience@verve.com
+LABEL version.spark=3.0.1
```

Comment:
In this doing something? Our spark version for data don't depend on Jupyter, but on the kernel mainly.

Reply:
not this just tells that the local installation on the image is 3.0.1 but it should be fine

Reply:
i put versions there so that in future you could simple do `docker image inspect ` to get some metadata on the image that might be useful

Reply:
👍